bitcoin/src/blockdata/script/push_bytes.rs

```diff
@@ -71,7 +71,7 @@ mod primitive {
         }
 
         /// Returns the underlying bytes.
-        pub fn as_bytes(&self) -> &[u8] { &self.0 }
+        pub const fn as_bytes(&self) -> &[u8] { &self.0 }
```

Comment:
I'm not a fan of this one either, none of the other `as_bytes` functions in the codebase are const.